### ISSUE TITLE
RDBMS Results/Query Window Unable to Resize

### ISSUE THREAD

**Issue by mmacny** [cloudera/hue#348]
Hi Team -

The UI for querying Hive tables is strong. The query editor is able to expand if SQL queries are many lines.

This is unfortunately not the case for the RDBMS Query Editor. It has many of our users thinking it's unusable in it's current state, but I assume there are some behind the scenes challenges at play. Any info on this?

**Comment by romainr** [comment]
Yes, DB Query app is way behind and is actually moving away in favor of the new generic editor, e.g. demo: 
http://gethue.com/custom-sql-query-editors/

In Hue 4 it will be officially all unified, and all the SQL editors will share 99% of the same good code.

**Comment by mmacny** [comment]
Thanks romainr -

One last question - What about downloading results from RDBMS queries? I'm running the latest master branch and my team members seem to only find a DL/Export button on Hive queries, not RDBMS queries.

Any help?

**Comment by romainr** [comment]
The issue is that there is no pagination available in these connector. We could add an option to download but limit at 1000 or 10000 rows for example. Do your users expect more rows?

**Comment by mmacny** [comment]
Ah, that makes sense. That config option for rdbms queries would definitely make this the only tool our team would need (10k would be great, but any exporting is a plus). If you know some hacks that allow us to add it ourselves let me know, or feel free to suggest how I can help contribute?

**Comment by romainr** [comment]
Quick way to show the status, change 'success' to 'available'
https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/static/notebook/js/notebook.ko.js#L621
(to be cleaner, we could return a status directly from the execute call, but I could help on that at the end)

The core of the work is in:
https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/rdbms.py#L104

to do something similar to:
https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/hiveserver2.py#L285

Basically, would be similar to the execute call. 

AFAIK the main issue is that there is no pagination, so if the call is big, it returns all the rows in one shot (connectors libs are https://github.com/cloudera/hue/tree/master/desktop/libs/librdbms/src/librdbms/server). Maybe we can leave for it for now, or see for automatically adding a LIMIT N to the queries with no limit in the future

**Comment by rainysia** [comment]
You can try to add the below  options  in your RDBMS options
`options={"init_command": "SET sql_select_limit=100; SET names 'utf8'"}`
It will set 100 row for the default limit row.

**Comment by romainr** [comment]
Created https://issues.cloudera.org/browse/HUE-3996

**Comment by romainr** [comment]
Fixed
